initializers/view.go

```diff
@@ -0,0 +1,51 @@
+package initializers
+
+import (
+	"go-google-scraper-challenge/helpers"
+	"html/template"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/beego/beego/v2/server/web"
+)
+
+// SetUpTemplateFunction register additional template functions
+func SetUpTemplateFunction() {
+	err := web.AddFuncMap("titlecase", strings.Title)
+	if err != nil {
+		log.Fatal("Failed to add template function", err.Error())
+	}
+
+	err = web.AddFuncMap("sentencecase", helpers.ToSentenceCase)
```

Comment:
Should we change the `sentencecase` to `sentence_case` to make it consistent with `render_file/icon`?